src/api/app/views/webui2/webui/project/side_links/_maintenance_project.html.haml

```diff
@@ -0,0 +1,14 @@
+%li
+  - if open_maintenance_incidents
+    %i.fa.fa-check-circle.text-success
+  - else
+    %i.fa.fa-exclamation-circle.text-danger
+  = link_to(project_maintenance_incidents_path(project)) do
+    = pluralize(open_maintenance_incidents.count, 'open incident')
```

Comment:
This is an array! Using `count` without block is really inefficient :see_no_evil:  Please user `size` or `length` :pray: